### ISSUE TITLE
Sequence contains no matching element error when running VS as Administrator

### ISSUE THREAD

**Issue by katit** [ErikEJ/EntityFramework6PowerTools#54]
VS 2017
When using "View entity data model(view-only)" feature I get popup with error "Sequence contains no matching element".

Tried to simplify solution as much as possible and narrowed down to the fact if I run solution by "double-click" on .sln - it works. However, when I run VS using my shortcut (in Admin mode) - I get this error.

**Comment by ErikEJ** [comment]
Are you able to share a repro?

**Comment by katit** [comment]
This is shortcut config. I wonder if it's related to "Start in" root. I use this shortcut to launch VS

Let me know if you need actual project. 


![vs shortcut](https://user-images.githubusercontent.com/8389381/54046014-bfc98400-4198-11e9-9f93-32b0662b9bd1.JPG)

**Comment by ErikEJ** [comment]
I think I need a project..

**Comment by katit** [comment]
Well. I started to come up with a project and couldn't reproduce it. Now I narrowed this down to how I open that specific solution. It is not related to Admin rights it seems, just to how solution opened.

When I open solution via File->Open Project/Solution it works
When I go to File->Recent Projects/Solutions - it doesn't work for that specific solution/location. For exact same solution moved to different location it works.

So it seems to be related to how link to "recent" stored in VS2017. And I don't think I can create reproduction.

Is there any way to enable logging/debugging so I can get you more details on when it happens? I am willing to share my PC via TeamViewer and demonstrate problem.
